src/v2/styles/_theme-variables.scss

```diff
@@ -1,3 +1,9 @@
+// Since Angular uses dynamic components it's not enough the hide the message input for theme-v1/v2, we have to remove it from the DOM tree
+// Angular component lifecycle requires us to place this on the root level
+:root {
```

Comment:
Not happy to add a root level variable but couldn't find a better solution (reason described above in the source code)